lib/v2/datasource/remote/model/profile_model.dart

```diff
@@ -1,28 +1,28 @@
 class ProfileModel {
-  final String? account;
-  final String? status;
-  final String? type;
-  final String? nickname;
-  final String? image;
-  final String? story;
-  final String? roles;
-  final String? skills;
-  final String? interests;
-  final int? reputation;
-  final int? timestamp;
+  final String account;
+  final String status;
+  final String type;
+  final String nickname;
+  final String image;
+  final String story;
+  final String roles;
+  final String skills;
+  final String interests;
+  final int reputation;
+  final int timestamp;
 
   ProfileModel({
-    this.account,
-    this.status,
-    this.type,
-    this.nickname,
-    this.image,
-    this.story,
-    this.roles,
-    this.skills,
-    this.interests,
-    this.reputation,
-    this.timestamp,
+    required this.account,
+    required this.status,
+    required this.type,
+    required this.nickname,
+    required this.image,
```

Comment:
is the image required? 
What if the user didnt upload an image? 

The image is not something that we force the user to submit when creating an account.

Reply:
Seems all the fields have a deault value from the back-end strigns have a empty strings